examples/syscall-steal.c

```diff
@@ -43,6 +43,15 @@
 
 #if defined(CONFIG_KPROBES)
 #define HAVE_KPROBES 1
+#if defined(CONFIG_X86_64)
+/* For x86 architecture, syscall table can no longer be used to invoke a syscall
```

Comment:
Rewrite the comment as following:
```c
 /* For x86 architecture, the syscall table cannot be used to invoke a syscall
 * after commit 1e3ad78.
 * Use a hook on the syscall entry instead to intercept the syscall.
 * Alternatively, set USE_KPROBES_PRE_HANDLER_BEFORE_SYSCALL to 0 to use the old method.
 */
```

Reply:
Adding the description of the kernel version, i.e., `... after v6.9 (commit 1e3ad78) ...`, would be better.